src/main/java/com/minecolonies/coremod/entity/ai/citizen/lumberjack/EntityAIWorkLumberjack.java

```diff
@@ -243,7 +295,7 @@ private IAIState prepareForWoodcutting()
     {
         if (checkForToolOrWeapon(ToolType.AXE))
         {
-            return getState();
+            return START_WORKING; // Reset everything, maybe there are new crafting requests
```

Comment:
comment on previous line please (also for line 314)

Reply:
Fixed in f1bab0e.